### DISCUSSION TITLE
在blender 4.2 中安装 prorender  出错 ！

### DISCUSSION THREAD

**Discussion by 13612810547** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#697] [Q&A]
目前  prorender 还不支持blender4.2版本  安装后会出现无法启用的问题， 啥时候能跟新支持到4.2 版本
<img width="627" alt="Snipaste_2024-11-25_23-28-51" src="https://github.com/user-attachments/assets/ec85808c-a757-446b-8349-95f3d6b5d5c2">

**Comment by takahiroharada** [comment]
Done.
